src/main/java/io/vertx/core/http/HttpClientRequest.java

```diff
@@ -389,6 +389,7 @@ default void send(ReadStream<Buffer> body, Handler<AsyncResult<HttpClientRespons
       setChunked(true);
     }
     body.pipeTo(this);
+    response(handler);
```

Comment:
can you set the response handler before calling `pipeTo` ?

Reply:
done. thank you for the review.